From 50739994efc659759eac55fbc113dcb26efb9479 Mon Sep 17 00:00:00 2001
From: Rico Ganahl <rico.ganahl@bytesatwork.ch>
Date: Wed, 14 Jun 2023 11:59:42 +0200
Subject: [PATCH] configs: bytedevkit: add flash manufacturers

Give the freedom to assembly SPI flash from different manufacturers.

Signed-off-by: Rico Ganahl <rico.ganahl@bytesatwork.ch>
---
 configs/bytedevkit_defconfig      | 3 +++
 configs/bytedevkit_fspi_defconfig | 3 +++
 2 files changed, 6 insertions(+)

diff --git a/configs/bytedevkit_defconfig b/configs/bytedevkit_defconfig
index 6f7027367a..1a50e4fcd7 100644
--- a/configs/bytedevkit_defconfig
+++ b/configs/bytedevkit_defconfig
@@ -65,9 +65,12 @@ CONFIG_DM_SPI_FLASH=y
 CONFIG_SF_DEFAULT_MODE=0
 CONFIG_SF_DEFAULT_SPEED=50000000
 CONFIG_SPI_FLASH_BAR=y
+CONFIG_SPI_FLASH_ATMEL=y
 CONFIG_SPI_FLASH_ISSI=y
 CONFIG_SPI_FLASH_MACRONIX=y
+CONFIG_SPI_FLASH_SPANSION=y
 CONFIG_SPI_FLASH_STMICRO=y
+CONFIG_SPI_FLASH_SST=y
 CONFIG_SPI_FLASH_WINBOND=y
 CONFIG_PHYLIB=y
 CONFIG_DM_ETH=y
diff --git a/configs/bytedevkit_fspi_defconfig b/configs/bytedevkit_fspi_defconfig
index 10c77bc817..50e6033aec 100644
--- a/configs/bytedevkit_fspi_defconfig
+++ b/configs/bytedevkit_fspi_defconfig
@@ -65,9 +65,12 @@ CONFIG_DM_SPI_FLASH=y
 CONFIG_SF_DEFAULT_MODE=0
 CONFIG_SF_DEFAULT_SPEED=50000000
 CONFIG_SPI_FLASH_BAR=y
+CONFIG_SPI_FLASH_ATMEL=y
 CONFIG_SPI_FLASH_ISSI=y
 CONFIG_SPI_FLASH_MACRONIX=y
+CONFIG_SPI_FLASH_SPANSION=y
 CONFIG_SPI_FLASH_STMICRO=y
+CONFIG_SPI_FLASH_SST=y
 CONFIG_SPI_FLASH_WINBOND=y
 CONFIG_PHYLIB=y
 CONFIG_DM_ETH=y
-- 
2.39.2

